src/opencensus_ext_newrelic/stats.py

```diff
@@ -59,8 +61,8 @@ class NewRelicStatsExporter(object):
         >>> stats_exporter.stop()
     """
 
-    def __init__(self, insert_key, service_name, host=None, interval=5):
-        client = self.client = MetricClient(insert_key=insert_key, host=host)
+    def __init__(self, insert_key, service_name, interval=5, host=None, port=None):
```

Comment:
The `port` should default to 443. We should probably have a test exposing this issue.
```suggestion
    def __init__(self, insert_key, service_name, interval=5, host=None, port=443):
```